ts/screens/wallet/payment/PaymentOutcomeCodeMessage.tsx

```diff
@@ -54,12 +60,19 @@ const successFooter = (onClose: () => void) => (
 const PaymentOutcomeCodeMessage: React.FC<Props> = (props: Props) => {
   const outcomeCode = props.outcomeCode.outcomeCode.fold(undefined, oC => oC);
 
+  const totalAmount =
+    (props.verifica.value.importoSingoloVersamento as number) +
```

Comment:
`verifica` and `psps` are of type `Pot` so should be handled properly [here](https://github.com/pagopa/io-app/blob/master/ts/features/bonus/bonusVacanze/components/RequestBonus.tsx#L73) an example